config/config-sample.js

```diff
@@ -20,12 +20,24 @@ module.exports = {
     user: '13466',
     password: '6pKF4jkv97zmqBJ3ZL8gUw5DfT2NMQ'
   },
+  auth_method:"static",
```

Comment:
Here I'd prefer:
auth: {
  method: 'static',
  static: { ...},
  confluence: {...}
}

Reply:
Also note the usage of ' instead of ". Doesn't matter functionally but nicer to keep style constant.